### PR TITLE
Reorganize cachito to use the web and workers subpackages

### PR DESCRIPTION
**Pull request by mprahl**
Reorganize cachito to use the web and workers subpackages and a couple small unrelated commits

### CODE REVIEW

**Comment by mprahl** [comment]
I'm fixing the tests now

File: cachito/web/splash.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from flask import render_template, Blueprint
 
 
-splash = Blueprint('splash', __name__, static_folder='static', template_folder='template')
```
Comment (lcarva) [inline cachito/web/splash.py]: Oh interesting! Not needed because it defaults to those values anyways?

File: requirements-workers.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+celery
```
Comment (lcarva) [inline requirements-workers.txt]: Don't you need celery also in the main `requirements.txt` ? I thought that was needed to actually create celery tasks.

File: cachito/web/splash.py (hunk 1 above)
Comment (mprahl) [reply cachito/web/splash.py]: Yup, it works just fine.

File: requirements-workers.txt (hunk 2 above)
Comment (mprahl) [reply requirements-workers.txt]: You're right. I'll fix it.

Comment (mprahl) [reply requirements-workers.txt]: It's actually not needed yet, so I'll just remove it from this PR and add it back later

**Comment by mprahl** [comment]
@lcarva this should be all set now.

**Pull request merged**